src/multivariate/mvnormal.jl

```diff
@@ -22,6 +22,51 @@
 #
 ###########################################################
 
+"""
+
+The [Multivariate normal distribution](http://en.wikipedia.org/wiki/Multivariate_normal_distribution)
+is a multidimensional generalization of the *normal distribution*. The probability density function of
+a d-dimensional multivariate normal distribution with mean vector ``\\boldsymbol{\\mu}`` and
```

Comment:
Hmmm, I'm not seeing that here, but looks like I did forget a couple extra backslashes on  `MvNormalCanon`.